benchmarks/db-benchmark/db-benchmark.dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y git build-essential
+
+# Install R, curl, and python deps
+RUN apt-get update && apt-get -y install --no-install-recommends --no-install-suggests \
```

Comment:
the apt-get update seems to be redundant? looks like we can merge these two apt runs to reduce build time.

Reply:
Good catch. Thx.